modules/currency-l0/src/main/scala/org/tessellation/currency/l0/cli/method.scala

```diff
@@ -172,8 +179,44 @@ object method {
         CollateralAmountOpts.opts,
         GlobalL0PeerOpts.opts,
         L0TokenIdentifierOpts.opts,
-        trustRatingsPathOpts
-      ).mapN(RunValidator.apply)
+        trustRatingsPathOpts,
+        lastKryoHashOrdinalOpts
+      ).mapN {
+        case (
+              storePath,
+              keyAlias,
+              password,
+              http,
+              environment,
+              snapshot,
+              seedlistPath,
+              prioritySeedlistPath,
+              collateralAmount,
+              globalL0Peer,
+              l0TokenIdentifier,
+              trustRatingsPath,
+              lastKryoHash
+            ) =>
+          val lastKH =
```

Comment:
I see it repeated many times. How about doing the following?

```scala
  val lastKryoHashOrdinalOpts: Opts[Option[SnapshotOrdinal]] =
    (
      AppEnvironment.opts,
      Opts
        .argument[NonNegLong]("lastKryoHashOrdinal")
        .map(SnapshotOrdinal(_))
        .orNone
    ).mapN {
      case (env, lastKryoHash) =>
        if (env === AppEnvironment.Dev) lastKryoHash else lastKryoHashOrdinal.get(env).orElse(SnapshotOrdinal.MinValue.some)
    }
```

Reply:
@kpudlik I've already fixed that once working on the configuration, so it makes no sense to fix it here again 🤔 I would have to rebase the configuration with these changes, so I can take a second look then once combining both changes. What do you think?